deploy/helm/templates/ingressroute.tpl

```diff
@@ -0,0 +1,19 @@
+apiVersion: traefik.containo.us/v1alpha1
```

Comment:
Missing {{- if .Values.ingress.enabled }} here?
IMO you probably want to make a `Values.ingress.traefik` block to gate this, as this is very specific to your setup.